### ISSUE TITLE
ProLIP instalation on colab

### ISSUE THREAD

**Issue by andresilvapimentel** [chemosim-lab/ProLIF#63]
My  colab notebook was working one month ago, but it stopped to work when the python 3.8 was required recently.
It seems the requirement to install Python 3.8+ in https://prolif.readthedocs.io/en/latest/source/installation.html does not work for colab as the colab default is python 3.7. However, I installed Python 3.8 and 3.9 on colab and did not work at all. It gave the same error when I installed Python 3.8 and 3.9 on Jupyter and Linux.

The only solution was using python 3.7 with:

!pip install --upgrade MDAnalysis
!pip install rdkit-pypi prolif

But I got the error message:

AttributeError                            Traceback (most recent call last)
<ipython-input-26-f55dc7e9c63e> in <module>()
      5 
      6 # create RDKit-like molecules for visualisation
----> 7 lmol = prolif.Molecule.from_mda(lig)
      8 pmol = prolif.Molecule.from_mda(prot)
      9 

AttributeError: module 'prolif' has no attribute 'Molecule'

I tried to solve the issue with the RDkit install, but nothing seems to work it. Do you have any suggestion?

**Comment by cbouy** [comment]
Hi @andresilvapimentel 

Colab is currently only available with python 3.7 but the most recent version of ProLIF available on PyPI (v1.0.0) requires python 3.8, and there doesn't seem to be a straightforward way to install python 3.8 on Colab at the moment.

> The only solution was using python 3.7 with:
!pip install --upgrade MDAnalysis
!pip install rdkit-pypi prolif

This will install a very (very) early version of ProLIF that completely differs from the one described in the published article and I don't recommend using it. 
If you want to install a recent version of ProLIF ([v0.3.4](https://prolif.readthedocs.io/en/v0.3.4/source/installation.html)) that should be compatible with Colab you could try:
```
!pip install rdkit-pypi cython
!pip install https://github.com/chemosim-lab/ProLIF/archive/v0.3.4.zip
```
This will compile MDAnalysis on Colab so you might run into some problems here but I think in another issue some people managed to make it work so in principle it should be doable.

Best,
Cedric

**Comment by andresilvapimentel** [comment]
I installed Python 3.8 and 3.9 on colab and did not work at all. It gave the same error when I installed Python 3.8 and 3.9, but  the installation procedure on https://prolif.readthedocs.io/en/latest/source/installation.html does not work for colab, Jupyter or Linux at all.
